openapi.yaml

```diff
@@ -0,0 +1,260 @@
+          description: FEDACH Financial Institution State
+        - name: city
+          in: query
+          schema:
+            type: string
+            example: 325183657
+          description: FEDACH Financial Institution City
+        - name: postalCode
+          in: query
+          schema:
+            type: string
+            example: 19753
+          description: FEDACH Financial Institution Postal Code
+        - name: limit
+          in: query
+          schema:
+            type: integer
+            example: 499
+          description: Maximum results returned by a search
+      responses:
+        '200':
+          description: FEDACH Participants returned from a search
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ACHDictionary'
+  /search/FEDWIRE:
+    get:
+      tags:
+        - FEDWIRE
+      summary: Search FEDWIRE names and metadata
+      operationId: searchFEDWIRE
+      parameters:
+        - $ref: '#/components/parameters/requestId'
+        - name: name
+          in: query
+          schema:
+            type: string
+            example: First Bank
+          description: FEDWIRE Financial Institution Name
+        - name: routingNumber
+          in: query
+          schema:
+            type: string
+            example: 324172465
+          description: FEDWIRE Routing Number for a Financial Institution
+        - name: state
+          in: query
+          schema:
+            type: string
+            example: PA
+          description: FEDWIRE Financial Institution State
+        - name: city
+          in: query
+          schema:
+            type: string
+            example: 325183657
+          description: FEDWIRE Financial Institution City
+        - name: limit
+          in: query
+          schema:
+            type: integer
+            example: 499
+          description: Maximum results returned by a search
+      responses:
+        '200':
+          description: FEDWIRE Participants returned from a search
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WIREDictionary'
+
+components:
+  schemas:
+    ACHDictionary:
+      description: Search results containing ACHDictionary of Participants
+      properties:
+        ACHPartcipants:
+          type: array
+          items:
+            $ref: '#/components/schemas/ACHParticipant'
+    ACHParticipant:
+      description: ACHParticipant holds a FedACH dir routing record as defined by Fed ACH Format.  https://www.frbservices.org/EPaymentsDirectory/achFormat.html
+      properties:
+        RoutingNumber:
+          type: string
+          description: The institution's routing number
+          example: 231380104
+        OfficeCode:
+          type: string
+          description: Main/Head Office or Branch. O=main B=branch
+          example: O=main B=branch
+        ServicingFrbNumber:
```

Comment:
Typically JSON names start lowercase and use camel case, so this would be `servicingFRBNumber`.